lms/djangoapps/ccx/api/v0/serializers.py

```diff
@@ -52,3 +54,10 @@ def get_course_modules(obj):
         Getter for the Course Modules. The list is stored in a compressed field.
         """
         return obj.structure or []
+
+    @staticmethod
+    def get_custom_content(obj):
```

Comment:
I think this is not necessary, you could use a DictField, why do you need this? @JuanDavidBuitrago

Reply:
@Squirrel18 I have updated the serializer to remove the static method ``get_custom_content``, as ``other_course_settings`` is a ``JSONField`` in the model that already handles default values and ensures it will always return a dictionary. Thanks for the suggestion!